src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  const addNumber = function(a) {
+    counter++;
+
+    return (b) => (counter < 4 || counter % 2)
+      ? a + b
+      : 'Bzzz... Error!';
+  };
+
+  return addNumber;
```

Comment:
you can рere return function

```suggestion
  return function(a) {
    counter++;

    return (b) => (counter < 4 || counter % 2)
      ? a + b
      : 'Bzzz... Error!';
  };
```